src/style.css

```diff
@@ -1 +1,122 @@
-/* empty source */
+/* #region general */
+html {
+  font-family: "Roboto", Arial, sans-serif;
+  font-weight: 400;
+  line-height: 14px;
+  font-size: 10px;
+  color: #616070;
+}
+
+body {
+  margin: 0;
+}
+/* #endregion */
+
+/* #region card */
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+/* #endregion */
+
+/* #region productInfo */
+.card__image {
```

Comment:
card elements are also a part of card block

Reply:
@mgrinko Do you mean for the region group?